### PR TITLE
DO-2496 made person detail view page

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

File: src/components/pages/datasets/person/const.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  ...asyncBlocks,
+  ...pepBlocks,
+  COINCIDENCE_BLOCK: 'COINCIDENCE_BLOCK',
+  ADDITIONAL_INFORMATION_BLOCK: 'ADDITIONAL_INFORMATION_BLOCK',
+};
+
+export const SOURCE = {
+  pep_source: { title: 'pep_ukr', translation: 'pepSource' },
+  sanction_source: { title: 'sanction_ukr', translation: 'sanctionSource' },
+};
```
Comment (roman-tiukh) [inline src/components/pages/datasets/person/const.js:13]:
```suggestion
export const SOURCE = {
  pep_ukr: { translation: 'pepSource' },
  sanction_ukr: { translation: 'sanctionSource' },
};
```

File: src/components/pages/datasets/person/utils.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { SOURCE } from './const';
+
+export const checkSource = (data) => {
+  const source = Object.values(SOURCE).find((item) => new RegExp(item.title).test(data.source));
+  return source.translation;
+};
```
Comment (roman-tiukh) [inline src/components/pages/datasets/person/utils.js:6]: 1. throw Error if source not found.
2. Why RegExp?

**Review by roman-tiukh — DISMISSED** [review]

File: src/components/pages/datasets/person/utils.js (hunk 2 above)
Comment (EmmaSamoilova) [reply src/components/pages/datasets/person/utils.js]: done

File: src/components/pages/datasets/person/const.js (hunk 1 above)
Comment (EmmaSamoilova) [reply src/components/pages/datasets/person/const.js]: done

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**